include/Extrema.h

```diff
@@ -54,22 +54,22 @@ struct Extrema_t
    Extrema_t()
    {
 
-      Field     = -1;
+      Field     = _NONE;
```

Comment:
Since `Macro.h` is not included in this file, I think `_NONE` is not defined yet.

Reply:
@hfhsieh Fixed